tests/Application.Tests.Integration/BaseClassFixture.cs

```diff
@@ -207,8 +207,8 @@ protected static Borrow CreateBorrowRequest(User borrower, Document document, Bo
             Document = document,
             Reason = "something something",
             Status = status,
-            BorrowTime = LocalDateTime.FromDateTime(DateTime.UtcNow),
-            DueTime = LocalDateTime.FromDateTime(DateTime.UtcNow + TimeSpan.FromDays(1))
+            BorrowTime = LocalDateTime.FromDateTime(DateTime.Now),
+            DueTime = LocalDateTime.FromDateTime(DateTime.Now + TimeSpan.FromDays(1))
```

Comment:
I really don't think using DateTime.Now is a good practice, but we'll investigate later